logger/log_writer.go

```diff
@@ -2,40 +2,22 @@ package logger
 
 import (
 	"encoding/json"
-	"os"
 
 	"github.com/sirupsen/logrus"
 )
 
 type LogWriter struct {
-	Logger   *logrus.Logger
-	Output   *os.File
-	LogLevel string
+	Logger *logrus.Logger
 }
 
 func (lw *LogWriter) Write(data []byte) (n int, err error) {
 	logFields := make(map[string]interface{})
 	err = json.Unmarshal(data, &logFields)
 	if err != nil {
-		Log.Warn("Unmarshalling Error in LogWriter: " + err.Error())
+		lw.Logger.Warn("Unmarshalling Error in LogWriter: " + err.Error())
 		return 0, err
 	}
 
-	lw.logByLevel(logFields)
+	lw.Logger.WithFields(logFields).Info()
 	return len(data), nil
 }
```

Comment:
I saw this is used in the middleware logger. Will we always log it at the `Info()` level?